crates/nu-utils/src/locale.rs

```diff
@@ -1,6 +1,5 @@
 use num_format::Locale;
 
-#[cfg(debug_assertions)]
```

Comment:
This was causing `cargo bench` to fail because the `nu!` macro uses this without checking `#[cfg(debug_assertions)]`, easiest fix was to delete the `#[cfg(debug_assertions)]`

Reply:
wow, that's a sneaky little thing. Glad you found it.

Reply:
Shouldn't change how locale works; just fixes a compile failure when running `cargo bench`. The local code scares me 😅